<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BindingProperties</class>
 <widget class="QWidget" name="BindingProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>602</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="DataComboBox" name="type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelEditor">
       <property name="text">
        <string>Editor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="DataComboBox" name="editor"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelTexture">
       <property name="text">
        <string>Texture</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ReferenceComboBox" name="texture"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelMagFilter">
       <property name="text">
        <string>Mag. Filter</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="DataComboBox" name="magFilter"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelMinFilter">
       <property name="text">
        <string>Min. Filter</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="DataComboBox" name="minFilter"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelAnisotropic">
       <property name="text">
        <string>Anisotropic</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="anisotropic"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelWrapModeX">
       <property name="text">
        <string>Wrap X</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="DataComboBox" name="wrapModeX"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelWrapModeY">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="DataComboBox" name="wrapModeY"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelWrapModeZ">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="DataComboBox" name="wrapModeZ"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelBorderColor">
       <property name="text">
        <string>Border</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="ColorPicker" name="borderColor"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="labelComparisonFunc">
       <property name="text">
        <string>Comparison</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="DataComboBox" name="comparisonFunc"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="labelLevel">
       <property name="text">
        <string>Level</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="level"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="labelLayer">
       <property name="text">
        <string>Layer</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QWidget" name="layerWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="layered">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="layer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="labelImageFormat">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="DataComboBox" name="imageFormat"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="labelBuffer">
       <property name="text">
        <string>Buffer</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="ReferenceComboBox" name="buffer"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="labelBlock">
       <property name="text">
        <string>Block</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="ReferenceComboBox" name="block"/>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="labelColor">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="ColorPicker" name="color"/>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="labelExpressions">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="ExpressionMatrix" name="expressions">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::CurrentChanged</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::DotLine</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>1</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="labelSubroutine">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QComboBox" name="subroutine">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataComboBox</class>
   <extends>QComboBox</extends>
   <header>session/DataComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ReferenceComboBox</class>
   <extends>QComboBox</extends>
   <header>session/ReferenceComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ExpressionMatrix</class>
   <extends>QTableWidget</extends>
   <header>session/ExpressionMatrix.h</header>
  </customwidget>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QCheckBox</extends>
   <header>session/ColorPicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
